README.md

```diff
@@ -52,3 +52,30 @@ Once you have the site set up locally, these are some common commands you might
 | Chrome / Android Web view | 44              | _Latest version with >0.5% of traffic_ |
 | Firefox                   | 52              | _Latest version with >0.5% of traffic_ |
 
+## Self-signed certificates
```

Comment:
Is this needed in the main README file?

If so, it might make more sense to have a quick link to the `certs/README.md` file to avoid double documentation.

Reply:
Also, presumably you all are still aiming to centralize documentation in the doc site, so that might be the better place for this explanatory info (which is the sort of thing we intentionally removed from the README a few months ago).

Reply:
@jbalboni where is the doc site? and i can remove this from the readme

Reply:
https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/ (https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/tree/master/packages/documentation)

Reply:
Ooh, good call @jbalboni. I don't personally know where we stand with that right now, but maybe @rianfowler can chime in and confirm that that's the right way to go.